.github/workflows/ci.yml

```diff
@@ -51,9 +80,15 @@ jobs:
         uses: ./.github/actions/setup-repo
         with:
           node-version: 18
-      
-  build:
-    needs: install
+      # Lint Styles
+      - name: Lint Styles
+        uses: ./.github/actions/run-script
+        with:
+          script-name: "lint:style"
+          concurrency: 10
```

Comment:
is this limiting to 10 concurrent actions?

Reply:
Yep! Same functionality that we had before